build-scripts/generate_man_page.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+
+import argparse
+import sys
+import ssg.build_profile
+import ssg.constants
+import ssg.xml
+import os
+
+import xml.etree.ElementTree as ET
+
+prolog = """
+.TH scap-security-guide 8 "26 Jan 2013" "version 1"
+
```

Comment:
We will need to update also this text.

Reply:
IMO it would be much better to have a jinja2 template for this outside of the script, it would make it much easier to edit and much easier to add more logic